src/plugin/markers/index.js

```diff
@@ -213,9 +228,11 @@ export default class MarkersPlugin {
         });
 
         const line = document.createElement('div');
+        const width = markerElement ? markerElement.width : this.markerWidth;
+        marker.offset = width / 2 - 0.5;
```

Comment:
can you move this 0.5 value to a meaningful variable?

Reply:
I moved to this.markerLineWidth/2

Reply:
what was 0.5 used for? And why is it removed?

Reply:
The 0.5 is the result of the half of marker position line width, and it's used to adjust the marker element in the center of this line.

I moved the hardcoded `1px` to `this.markerLineWidth` and replaced the 0.5 for `markerLineWidth/2`.

Previously we have:
```
        this.style(line, {
            "flex-grow": 1,
            "margin-left": (this.markerWidth / 2 - 0.5) + "px",
            background: "black",
            width: "1px",
            opacity: 0.1
        });
```

After the creation of `markerLineWidth` var we can replace it for:
```
        this.style(line, {
            "flex-grow": 1,
            "margin-left": (this.markerWidth / 2 - this.markerLineWidth / 2) + "px",
            background: "black",
            width: this.markerLineWidth + "px",
            opacity: 0.1
        });
```

After that, we can simplify the expression  for `margin-left`: 

```
        this.style(line, {
            "flex-grow": 1,
            "margin-left": (this.markerWidth - this.markerLineWidth ) / 2 + "px",
            background: "black",
            width: this.markerLineWidth + "px",
            opacity: 0.1
        });
```

Finally, I set this value in  `market.offset` because is reutilized in `_updateMarkerPosition`:

```
        marker.offset = (width - this.markerLineWidth) / 2;
        this.style(line, {
            "flex-grow": 1,
            "margin-left": marker.offset + "px",
            background: "black",
            width: this.markerLineWidth + "px",
            opacity: 0.1
        });
```
I can just put the value in a variable named `halfLineWidth` if you think that it's more easy to read.